WooCommerce/Classes/Universal Links/UniversalLinkRouter.swift

```diff
@@ -7,14 +7,18 @@ import UIKit
 struct UniversalLinkRouter {
     private let matcher: RouteMatcher
     private let bouncingURLOpener: URLOpener
+    private let analyticsTracker: UniversalLinkRouterAnalyticsTracking?
 
     /// The order of the passed Route array matters, as given two routes that handle a path only the first
     /// will be called to perform its action. If no route matches the path it uses the `bouncingURLOpener` to
     /// open it e.g to be opened in web when the app cannot handle the link
     ///
-    init(routes: [Route], bouncingURLOpener: URLOpener = ApplicationURLOpener()) {
+    init(routes: [Route],
+         bouncingURLOpener: URLOpener = ApplicationURLOpener(),
+         analyticsTracker: UniversalLinkRouterAnalyticsTracking? = UniversalLinkAnalyticsTracker(analytics: ServiceLocator.analytics)) {
```

Comment:
One question here, why do we create this `UniversalLinkRouterAnalyticsTracking` and inject it here instead of just using the `ServiceLocator.analytics` as before?